ticdc/troubleshoot-ticdc.md

```diff
@@ -257,6 +257,10 @@ In TiCDC Open Protocol, the type code `6` represents `null`.
 
 For more information, refer to [TiCDC Open Protocol column type code](/ticdc/ticdc-open-protocol.md#column-type-code).
 
+## The `start-ts` timestamp of the TiCDC is quite different from the current time. During the execution of the task, replication is interrupted and an error `[CDC:ErrBufferReachLimit]` occurs
+
+Since v4.0.9, you can try to enable the `unified sorter` feature for replication, or use the BR tool for an incremental backup and recovery, and then start the TiCDC replication task from a new point in time. TiCDC will optimize this issue in future versions.
```

Comment:
```suggestion
Since v4.0.9, you can try to enable the unified sorter feature in your replication task, or use the BR tool for an incremental backup and restore, and then start the TiCDC replication task from a new time. This issue might be fixed in future versions.
```